source/documentation/deploying_services/mysql.md

```diff
@@ -0,0 +1,222 @@
+
+GOV.UK PaaS will automatically parse the ``VCAP_SERVICES`` [environment variable](/#system-provided-environment-variables) to get details of the  service and then set the `DATABASE_URL` variable to the first database found.
+
+Use ``cf env APPNAME`` to see the app's environment variables and confirm that the variable has been set correctly.
+
+If your app writes database connection errors to `STDOUT` or `STDERR`, you can view recent errors with ``cf logs APPNAME --recent``. See the section on [Logs](#logs) for details.
+
+## MySQL maintenance & backups
+
+### MySQL service maintenance times
+
+The PaaS MySQL service is currently provided by Amazon Web Services RDS. Each MySQL service you create will have a randomly-assigned weekly 30 minute maintenance window, during which there may be brief downtime. (To minimise this downtime, select a high availability plan with `HA` in its name).
+
+Minor version upgrades (for example from 5.7.10 to 5.7.11) will be applied during the maintenance window.
+
+For more details, see the [Amazon RDS Maintenance documentation](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_UpgradeDBInstance.Maintenance.html) [external page].
+
+If you need to know the time of your maintenance window, please contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk). Window start times will vary from 22:00 to 06:00 UTC. We will add the ability to set the time of the maintenance window in a future version of GOV.UK PaaS.
+
+### MySQL service backup
+
+The data stored within any MySQL service you create is backed up using the standard Amazon RDS backup system, except if you are using the free plan.
+
+Backups are taken nightly at some time between 22:00 and 06:00 UTC. Data is retained for 7 days.
+
+There are two ways you can restore data to an earlier state:
+
+1. You can restore to the latest snapshot yourself. See [Restoring a MySQL service snapshot](#restoring-a-mysql-service-snapshot) for details.
+
+2. We can manually restore to any point from 5 minutes to 7 days ago, with a resolution of one second. Data can be restored to a new MySQL service instance running in parallel, or it can replace the existing service instance.
+
+    To arrange a manual restore, contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk). We will need approval from your organization manager if restoring will involve overwriting data.
+
+Note that data restore will not be available in the event of an RDS outage affecting the entire Amazon availability zone.
+
+For more details about how the RDS backup system works, see [Amazon's DB Instance Backups documentation](http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Overview.BackingUpAndRestoringAmazonRDSInstances.html) [external page].
+
+### Restoring a MySQL service snapshot (experimental)
```

Comment:
I think we should omit this section because we don't believe this works yet. We have a separate story to enable this for MySQL.